Tests/Bundle/BundleTest.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace League\Tactician\Bundle\Tests\Bundle;
+
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Symfony\Component\Yaml\Yaml;
+
+/**
+ * To ensure cache is isolated from each test.
+ *
+ * @runTestsInSeparateProcesses
+ */
+class BundleTest extends KernelTestCase
+{
+    protected function setUp()
+    {
+        static::$kernel = static::createKernel();
+        $dir = sys_get_temp_dir().DIRECTORY_SEPARATOR.'tactician-bundle'.DIRECTORY_SEPARATOR.md5(microtime(true) * rand(0, 10000));
+        if (false === is_dir($dir)) {
+            mkdir($dir, 0777, true);
+        }
+        static::$kernel->defineCacheDir($dir);
```

Comment:
It helps debug to have it at the end. As we clear on startup, I guess it is enough to prevent weird edge case ?